### DISCUSSION TITLE
default to importStrategy fromEnvironment if pylint is in available

### DISCUSSION THREAD

**Discussion by davetapley** [microsoft/vscode-pylint#558] [Ideas]
I was a bit confused just now because I bumped to the [new pylint 3.2](https://github.com/pylint-dev/pylint/releases/tag/v3.2.2), but was still seeing an erroneous error (from https://github.com/pylint-dev/astroid/issues/1015).

Only after some digging did I realize the extension comes bundled with a `pylint` version.
I see I can  `importStrategy` `fromEnvironment`, which is fine as a workaround, but it made me think:

Would a PR be welcome to change the behavior to default to `importStrategy` `fromEnvironment` if `pylint` is in available in the current environment? I can only assume that's what 99.99% of people would expect.

**Comment by Pierre-Sassoulas** [comment]
Neutral on this. I think experts can find and change the option 'easily' but non-experts / newcomers can easily shoot themself in the foot with a bad env and an incompatible pylint. Maybe it's a good thing that the default protect newcomers from 'breaking' the plugin.

**Comment by karthiknadig** [comment]
I agree with @Pierre-Sassoulas . We have seen new users struggle with env issues, hence the default to run with bundled.
